app/controllers/tenants/projects_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Tenants::ProjectsController < ApplicationController
+  include ProjectsHelper
+
+  def index
+    @tenant = Tenant.find_by(slug: params[:slug])
+
+    redirect_to root_path if @tenant.nil?
+
+    if params[:category_name] == 'Shop All' || params[:category_name].nil?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.